core/src/uix/linter.clj

```diff
@@ -260,10 +260,13 @@
   (read-re-frame-config))
 
 (defn lint-re-frame! [form env]
-  (let [sources (->> (uix.lib/find-form rf-subscribe-call? form)
-                     (keep #(let [v (ana/resolve-var env (first %))]
-                              (when (contains? re-frame-config (:name v))
-                                (assoc v :source %)))))]
+  (let [resolve-fn (if (uix.lib/cljs-env? env)
+                     ana/resolve-var
+                     resolve)
```

Comment:
would be good to add a basic test for the linter running on JVM, wdyt?

Reply:
Yeah nice idea. Is it fine if the failing cases just crashes the tests? There's no `is-not-thrown?` in Clojure test, and it felt quite heavy adding it just for this case

Reply:
Since all linters are throwing, it would be actually great to have such utility! We'll use for other tests as well

Reply:
Okay cool, I will add it!

Reply:
Tests are here https://github.com/pitch-io/uix/pull/92/commits/a3da57841bd3311207bb007821050a7c26db598c